### PR TITLE
Feat/zustandandlocalstorage

### PR DESCRIPTION
**Pull request by PhilippWeiss85**
Dear beloved reviewer,

in this PR the following changes happened:
- Added useStore/Zustand
- Switched  initial dummy array and setter functions to useStore
- Refactored the code

Thanks for reviewing!

### CODE REVIEW

File: store/useStore.js
Comment (kai-franke) [inline store/useStore.js:85]: I don't know if it's possible but for readability it would be very nice if this initial data was stored outside and imported here, right?

File: store/useStore.js
Comment (ThomasF85) [inline store/useStore.js:5]: Very nice use of zustand!
One thing to consider: You have the following states in your store: games, appendNewGameCard, deleteCard, updateCardDetail. You are mixing game and card in your naming and you are inconsistent. As you application gets bigger the naming gets more important because you want to not get confused by it. It's best to be consistent and decisive in your naming (for example: games, appendGame, deleteGame, updateGame).

**Review by ThomasF85 — APPROVED** [review]
Very nice! I just have a comment regarding the naming of your states. Feel free to incorporate the changes if you want and merge.

File: store/useStore.js
Comment (ThomasF85) [reply store/useStore.js:4]: Another thing is that you have not used localStorage here. The branch naming is confusing.

**Review by kai-franke — APPROVED** [review]
@PhilippWeiss85 , I love how you make use of Zustand and it inspires me to try that in my project now too.
I could only find one small thing that concerns readability. Since this step is certainly very important for your project, it might be good to have a second review from someone - if you feel like it, launch the rocket! :)

File: store/useStore.js
Comment (PhilippWeiss85) [reply store/useStore.js:5]: Hey Thomas, thanks I will change the names as you suggested

File: store/useStore.js
Comment (PhilippWeiss85) [reply store/useStore.js:85]: Hey Kai, thanks for the input. You are definitely right.. the initial data should be stored outside. Will do this change in a future PR

**Pull request merged**
